Source/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -1299,11 +1299,11 @@ - (UIBezierPath *)accessibilityPath
   return _getAccessibilityFromViewOrProperty(_accessibilityPath, accessibilityPath);
 }
 
-- (NSInteger)accessibilityElementCount
-{
-  _bridge_prologue_read;
-  return _getFromViewOnly(accessibilityElementCount);
-}
+//- (NSInteger)accessibilityElementCount
```

Comment:
Presumably delete this code, seems like it's now defined in the other category.

Reply:
Done

Reply:
Done